Automattic/jetpack
# Boost: Fix initial score

## Proposed changes:
On a fresh install like every other time, boost shows two scores (with and without boost). But, both scores are computed for the same state of the site, with no modules being in effect. This comparison is confusing. So, this PR changes the initial score measurement to only return a single score without `no-boost` score to avoid comparison.

### Other information:


## Jetpack product discussion
None

## Does this pull request change what data or activity we track or use?
None

## Testing instructions:
* Setup boost on a new site
* Make sure there is no `no-boost` score on initial load
* Make sure there is a `no-boost` score present if you dispatch a speed score request after enabling the modules.
![image](https://github.com/Automattic/jetpack/assets/3737780/d8627647-b3e8-4772-8816-6e5c60962cdd)
